capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] We are required to capitalize the correct_name, not make it an uppercase. Kindly replace `.upcase` with `.capitalize`.

Reply:
## Thank you @DammyShittu